outlier_detection_python.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+from scipy.ndimage import median_filter
+
+
+def outlier_detection_fieldmap(B0map, neighsz, thresh):
+    # Apply a median filter to the B0map directly
```

Comment:
I would replace the repetitive comments with a single numpy docstring (please check how to do it, its a good practice for documentations.